assets/js/impactform.js

```diff
@@ -0,0 +1,60 @@
+$('#input-feedback').hide();
+
+$('[data-toggle="datepicker"]').datepicker({
+    todayHighlight: true,
+    dateFormat: 'dd/mm/yyyy' 
+});
+
+    $('#input-form').one('submit',function(){
```

Comment:
didn't see that before, sorry.. can you move this to the left so we use the same indentation for everything?